### ISSUE TITLE
Do you want to replace __asm__ with __builtin_ctzll(b)?

### ISSUE THREAD

**Issue by scchess** [basil00/Fathom#14]
static inline unsigned lsb(uint64_t b)
{
    return __builtin_ctzll(b);    
}

**Comment by basil00** [comment]
Yes it could be changed.  I'll keep this issue open and make the update the next time I modify Fathom.
